src/components/app/details/triggerView/CiWebhookDebuggingModal.tsx

```diff
@@ -14,365 +14,285 @@
-                        >
-                            {/* Here the CI model requires the CiPipelineId not the CiPipelineMaterialId */}
-                            <Edit className=" icon-dim-24" />
-                        </button>
-                    </div>
-                    <div>
-                        <div
-                            className="cn-5 fw-6 pt-8 pb-8 dc__border-bottom"
-                            style={{ display: 'grid', gridTemplateColumns: '40% 20% 20% 20%', height: '100' }}
-                        >
-                            <div className="pl-8">Selector/Key</div>
-                            <div>Selector value in payload</div>
-                            <div>Configured filter</div>
-                            <div>Result</div>
-                        </div>
+    const getWebhookIncomingPayload = () =>
+        webhookIncomingPayload?.selectorsData?.sort((a, b) => sortCallback('selectorName', a, b))
 
-                        {webhookIncomingpayload?.map((selectedData, index) => {
-                            let classes = 'cn-7 pt-8 pl-4 pb-8'
-                            if (index % 2 == 0) {
-                                classes = 'cn-7 pt-8 pl-4 pb-8 bcn-1'
-                            }
-                            return (
-                                <div
-                                    key={index}
-                                    className={classes}
-                                    style={{ display: 'grid', gridTemplateColumns: '40% 20% 20% 20%', height: '100' }}
-                                >
-                                    <div>{selectedData?.selectorName}</div>
-                                    <div>{selectedData?.selectorValue}</div>
-                                    <div>{selectedData?.selectorCondition}</div>
-                                    <div
-                                        className={
-                                            selectedData?.match == false ? `dc__deprecated-warn-text` : `cg-5 ml-4`
-                                        }
-                                    >
-                                        {selectedData?.match === false ? 'Did not match' : 'Matched'}
-                                    </div>
-                                </div>
-                            )
-                        })}
+    const renderFilterTableContent = () => (
+        <div className="w-100">
+            <div className="cn-7 fw-6 dc__border-bottom ci__filter-table__row dc__uppercase py-8 fs-12 dc__gap-12">
+                <div className="lh-20">Selector/Key</div>
+                <div className="lh-20">Selector value in payload</div>
+                <div className="lh-20">Configured filter</div>
+                <div className="lh-20">Result</div>
+            </div>
+
+            {getWebhookIncomingPayload()?.map((selectedData: WebhookReceivedFiltersType) => (
```

Comment:
Please add a default value in the method and remove unnecessary `?` check from here and following lines